src/diffpy/srfit/tests/testbuilder.py

```diff
@@ -170,7 +170,7 @@ def testParseEquation(self):
         sigma = 0.1
         eq.x.setValue(x)
         eq.sigma.setValue(sigma)
-        f = lambda x, sigma: sqrt(e ** (-0.5 * (x / sigma) ** 2))
+        f = lambda x, sigma: sqrt(e ** (-0.5 * (x / sigma) ** 2))  # noqa: E731
```

Comment:
Here in the tests they are quite readable and not doing much harm so I suggest leaving with noqa